src/coreclr/vm/codeman.cpp

```diff
@@ -1827,7 +1839,11 @@ static void LoadAndInitializeJIT(LPCWSTR pwzJitName DEBUGARG(LPCWSTR pwzJitPath)
         EX_TRY
         {
             typedef void (* pjitStartup)(ICorJitHost*);
+#ifdef TARGET_WASM
```

Comment:
Similar here, we may want to introduce a named define for everything statically linked together. 

We have `FEATURE_MERGE_JIT_AND_ENGINE` to do this for the JIT that we may be able to reuse here (we can rename it to avoid confusion if it applies to both JIT and interpreter).